app/react/portainer/environments/ListView/columns/url.tsx

```diff
@@ -33,17 +33,17 @@ function Cell({
     return (
       <>
         {environment.URL}
-        {environment.StatusMessage.Summary &&
-          environment.StatusMessage.Detail && (
+        {environment.StatusMessage?.Summary &&
+          environment.StatusMessage?.Detail && (
             <div className="ml-2 inline-block">
               <span className="text-danger vertical-center inline-flex">
                 <AlertCircle className="lucide" aria-hidden="true" />
-                <span>{environment.StatusMessage.Summary}</span>
+                <span>{environment.StatusMessage?.Summary}</span>
```

Comment:
here and on line 46 `?` is not needed as because of lines 36-37 we know they're defined

Reply:
Good point 👍